unit_testing/tests/userDataHandler.test.js

```diff
@@ -0,0 +1,188 @@
+const UserDataHandler = require('../src/data_handlers/user_data_handler.js')
+const nock = require('nock')
+const userData = require('../data/users.json')
+
+const testUserData = userData[1]
+let userDataHandler
+
+function setupMock () {
+  if (process.env.MOCK) {
+    nock(global.SERVER_URL)
+      .get('/users')
+      .reply(200, userData)
+  }
+}
+
+describe('when users were not loaded', () => {
+  beforeEach(async () => {
+    setupMock()
+    userDataHandler = new UserDataHandler()
+  })
+
+  afterEach(() => {
+    nock.cleanAll()
+  })
```

Comment:
those could be top level hooks:
```suggestion
beforeEach(async () => {
  setupMock()
  userDataHandler = new UserDataHandler()
})

afterEach(() => {
  nock.cleanAll()
})

describe('when users were not loaded', () => {
```

Reply:
Agreed. Commited.